packages/material-ui/src/ToggleButton/ToggleButton.js

```diff
@@ -72,6 +105,15 @@ const ToggleButton = React.forwardRef(function ToggleButton(props, ref) {
     ...other
   } = props;
 
+  const styleProps = {
+    ...props,
+    disableFocusRipple,
+    selected,
+    size,
```

Comment:
```suggestion
    ...props,
    disabled,
    disableFocusRipple,
    size,
```

Reply:
Do we not need `selected` for `useUtilityClasses` `slots`?

Reply:
We need it and we have it as it is included in the props. It doesn't have any default value, so we don't have to specify it again.